hw7/task2/library.js

```diff
@@ -0,0 +1,206 @@
+const { setImmediate, setInterval } = require('timers');
+const repl = require('repl'); 
+class Library{        
+    Track = [];
+    Album = [];
+    Artist = [];
+    //singlton realization fir Library
+    constructor(){
```

Comment:
getInstance static method is not defined.

Reply:
fixed

Reply:
Not fixed.
`getInstance` should include checks defined in your constructor